pom.xml

```diff
@@ -29,7 +29,7 @@
     <maven.javadoc.skip>true</maven.javadoc.skip>
     <snakeyaml.version>1.26</snakeyaml.version>
     <java.level>8</java.level>
-    <jenkins.version>2.164.3</jenkins.version>
+    <jenkins.version>2.204.6</jenkins.version>
```

Comment:
2.190.1 would have done it, no?

Reply:
Most plugins seem to be taking 2.204 no real reason though for this one